src/Components/SmartComponents/Reports/Common/firstPagePDF.js

```diff
@@ -69,7 +69,7 @@ const firstPagePDF = ({ cves, meta, filters, intl, isReportDynamic, reportData,
             <Paragraph>
                 {intl.formatMessage(messages.CVEsPDFreportSubHeader, {
                     cve_count: meta.total_items.toString(),
-                    system_count: 100,
+                    system_count: meta.total_items,
```

Comment:
heh, using `meta.total_items` for both cve and system? 
check https://github.com/RedHatInsights/vulnerability-engine/pull/726/files#diff-81484eeb806817e4bfb29922596f290bR128

Reply:
ooops